servers/audio_server.cpp

```diff
@@ -216,6 +216,9 @@ void AudioDriverManager::initialize(int p_driver) {
 	GLOBAL_DEF_RST("audio/driver/enable_input", false);
 	GLOBAL_DEF_RST(PropertyInfo(Variant::INT, "audio/driver/mix_rate", PROPERTY_HINT_RANGE, "11025,192000,1,or_greater,suffix:Hz"), DEFAULT_MIX_RATE);
 	GLOBAL_DEF_RST(PropertyInfo(Variant::INT, "audio/driver/mix_rate.web", PROPERTY_HINT_RANGE, "0,192000,1,or_greater,suffix:Hz"), 0); // Safer default output_latency for web (use browser default).
+	GLOBAL_DEF_RST("audio/driver/override_channels", false);
+	GLOBAL_DEF_RST(PropertyInfo(Variant::INT, "audio/driver/override_speaker_channels", PROPERTY_HINT_RANGE, "2,8,1"), 8);
```

Comment:
Should mono configurations also be allowed? I'm not sure if you can still get a mono *output* on a PC intentionally in 2025. (For audio *input*, you can, e.g. if using a Bluetooth headset + microphone.)

Also, do configurations with 5 or 7 channels make sense? To my knowledge, this is what exists:

- Mono, 1 channel
- Stereo (2.0), 2 channels
- 2.1, 3 channel. Seems to be abstracted away from the software, e.g. a soundbar with a subwoofer besides it. In practice, the software sees a 2.0 setup.
- 5.1, 6 channels
- 7.1, 8 channels

If 5 and 7 channels don't actually exist in real audio setups, then the property could have an enum hint to specify which values are allowed (and give them a description):

```
PROPERTY_HINT_ENUM, "Stereo:2,Surround 5.1:6,Surround 7.1:8"
```

Reply:
I like this, but it's so far outside of my knowledge and could be misleading about what we have implemented.  What I propose now is simple and close to the C++ implementation.  If I do more of this I will have to some surround sound hardware to plug my PC into so it's not done completely blind.